orquesta/conducting.py

```diff
@@ -882,3 +885,40 @@ def get_task_transition_contexts(self, task_id, route):
                 contexts[task_transition_id] = self.get_task_initial_context(t[1], route)
 
         return contexts
+
+    def request_rerun_workflow_status(self, status, rerun_options):
+        # Reset workflow status.
+        current_status = self.get_workflow_status()
+        if current_status != statuses.FAILED:
+            LOG.debug('Workflow execution is not rerunable because its status is not Failed.')
```

Comment:
Please change the message to `Workflow execution cannot rerun from task(s) because it is not in a failed state.`